src/ui/map.js

```diff
@@ -1266,16 +1293,33 @@ class Map extends Camera {
      * });
      * @see [Example: Create a draggable point](https://docs.mapbox.com/mapbox-gl-js/example/drag-a-point/)
      */
-    off(type: MapEvent, layerId: any, listener: any) {
+    off(type: MapEvent, layerIds: any, listener: any) {
         if (listener === undefined) {
-            return super.off(type, layerId);
+            return super.off(type, layerIds);
+        }
+
+        if (!Array.isArray(layerIds)) {
```

Comment:
nit: negate the conditional since both branches are handled anyway, or, at risk of code-golfing,

```suggestion
        layerIds = new Set(Array.isArray(layerIds) ? layerIds : [layerIds]);
```

Reply:
I can set that 
```
map.off('mousedown', 'layer1', callback);
```

will work. Is that what you guys need?